traffic-policy/actions/oauth/config.mdx

```diff
@@ -0,0 +1,148 @@
+import { Config, ConfigItem } from "/src/components/ConfigTable.tsx";
+
+## Configuration Reference
+
+The [Traffic Policy](/traffic-policy/) configuration
+reference for this action.
+
+### Supported Phases
+
+`on_http_request`
+
+### Type
+
+`oauth`
+
+### Configuration Fields
+
+<Config>
+	<ConfigItem title="provider" type="string" required={true}>
+		<p>
+			The name of the _OAuth_ identity provider to be used for authentication.
+		</p>
+	</ConfigItem>
+	<ConfigItem title="auth_id" type="string" required={false}>
+		<p>
+			Unique authentication identifier for this provider. This value will be
+			used for the cookie, redirect, authentication and logout purposes.
+		</p>
+		<p>
+			To login a user you must use `/ngrok/login?auth_id={auth_id}`. If you are
+			using path based auth you must include the path to be redirected back to:
+			`?redirect_path=/foo`
+		</p>
+		<p>To log a user out you must use `/ngrok/logout?auth_id={auth_id}`</p>
+		<p>
+			You additionally can provide a path to redirect the user to on logout:
+			`?redirect_path=/`
+		</p>
```

Comment:
It feels like this should have it's own section. I know there is **Special Paths** below, but it feels like this needs something more than just these two sections. I'm envisioning something where it's **How do I log users in?** and **How do I log users out?** and they are linked from here. Perhaps both of these are captured in a single section and can be linked as a "Learn more about login and logout"

Reply:
how do you feel about opening a follow issue for this?

Reply:
created #1119